### PR TITLE
Added two new columns to upload table

### PR DESCRIPTION
**Pull request by johanna-skylight**
# DATABASE PULL REQUEST

## Related Issue

- First step for 6723

## Changes Proposed

- Add the new columns destination and disease to the upload table

## Testing
Check that the DB gets updated and and rollbacks work.

### CODE REVIEW

**Comment by johanna-skylight** [comment]
Just realize that some backend tests are failing with this change. Please don't review until I get that fixed.

**Comment by johanna-skylight** [comment]
I set the `supported_disease_id` column to accept null values to make it backwards compatible. I had previously set it as not nullable and this was causing the backend tests to break. This PR should be good again for review.

**Review by fzhao99 — APPROVED** [review]
lgtm!

File: backend/src/main/resources/db/changelog/db.changelog-master.yaml [hunk 1]
```diff
@@ -5096,4 +5096,34 @@ databaseChangeLog:
...
+                  name: supported_disease_id
+                  remarks: The disease that was reported.
+                  type: uuid
+                  constraints:
+                    nullable: true
+                    foreignKeyName: fk__upload__supported_disease_id
+                    referencedTableName: supported_disease
+                    referencedColumnNames: internal_id
+      rollback:
+        sql: |
```
Comment (emyl3) [inline backend/src/main/resources/db/changelog/db.changelog-master.yaml:5128]: in the rollback don't we want to drop the `supported_disease_id` column? 🤔

Comment (johanna-skylight) [reply backend/src/main/resources/db/changelog/db.changelog-master.yaml:5128]: My understanding is that [addColumn](https://docs.liquibase.com/change-types/add-column.html) has rollback automatically performed. But would love @BobanL to confirm this!

Comment (emyl3) [reply backend/src/main/resources/db/changelog/db.changelog-master.yaml:5128]: I rolledback locally and saw the `destination` column get dropped but the `supported_disease_id` remained -- not sure if anyone else can repro

Comment (emyl3) [reply backend/src/main/resources/db/changelog/db.changelog-master.yaml:5128]: let me try again as a sanity check 😅

Comment (fzhao99) [reply backend/src/main/resources/db/changelog/db.changelog-master.yaml:5128]: ah actually i do see the second `supported_disease_id` still after the rollback

Comment (johanna-skylight) [reply backend/src/main/resources/db/changelog/db.changelog-master.yaml:5128]: wonder if it is due to the constraints... I will add the drop column in the rollback section. Thanks for catching that @emyl3 !

Comment (emyl3) [reply backend/src/main/resources/db/changelog/db.changelog-master.yaml:5128]: I wonder if you can't mix and match auto rollback vs defining your own rollback https://docs.liquibase.com/workflows/liquibase-community/using-rollback.html?_ga=2.72264851.1204933114.1698177543-1193148286.1698177543 (see "Override default rollback commands" section) 🤔

**Review by fzhao99 — APPROVED** [review]
tested locally and rollback worked as expected!

**Review by emyl3 — APPROVED** [review]
LGTM!
